sentinelhub/constants.py

```diff
@@ -342,28 +342,22 @@ def from_string(mime_type_str):
         :return: A mime type enum
         :rtype: MimeType
         """
+        guessed_extension = mimetypes.guess_extension(mime_type_str)
+        mime_type_str = guessed_extension.strip('.') if guessed_extension else mime_type_str
+
```

Comment:
I would prefer if you left in that splitting by `'/'`. That would handle case `application/tar`.

I think this splitting should be important also to simplify [this line](https://github.com/sentinel-hub/sentinelhub-py/blob/master/sentinelhub/sentinelhub_request.py#L48)